app/routes/convert.py

```
@@ -462,6 +464,12 @@ def __init__(self, message: str):
         400,
         "PS3 folder contains symlinks or non-regular entries and cannot be packed safely",
     ),
+    SkipReason.SOURCE_COMPANION_UNSAFE: (
+        400,
+        "A companion file this source consumes is a symlink or resolves outside "
+        "the configured volumes (for a split Wii U dump, one of the other "
+        "game_partN.wud files)",
```

Ruff (0.16.1)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)